### PR TITLE
Simplify and remove redundant text in the Identity section.

### PR DESCRIPTION
**Pull request by jyasskin**
I haven't touched the Recognition Methods section in this PR.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/218.html#identity" title="Last updated on Feb 15, 2023, 5:20 PM UTC (e770f42)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/218/d9fd32a...jyasskin:e770f42.html" title="Last updated on Feb 15, 2023, 5:20 PM UTC (e770f42)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1049,11 +1057,6 @@
 party, you expect them to have different discussion topics with you than they
 usually would, and possibly even to pretend they don't know you.
 
-Note that a user agent may not always be powerful enough to prevent [=cross-context
-recognition=]. For example, if two or more services [=inappropriately=]
-require an identifying piece of information (eg. an email address) in order
-to use the services.
```
Comment (jyasskin) [inline index.html:1055]: The comment about user agent power didn't really fit in the "recognition types" section, and I moved the "identifying piece of information" up to where we say what cross-context recognition is appropriate.

File: index.html [hunk 2]
```diff
@@ -1093,49 +1094,16 @@
...
-instance by partitioning identities per subdomain or site path when the
-structure of a given site is known.
-
-Where possible, [=user agents=] should prevent people from being
-[=cross-context recognition|recognized=] across [=partitions=] unless they
-intend to be recognized.
-
-If a [=person=] visits two or more web pages from different [=partitions=],
-and does not explicitly express the same identity on each visit, the user agent should
-take steps to prevent [=same-site recognition=]. Note that sites can do harm
```
Comment (jyasskin) [inline index.html:1107]: Redundant with the previous paragraph.

File: index.html [hunk 3]
```diff
@@ -1093,49 +1094,16 @@
...
-
-This principle is limited in cases where preventing
-[=cross-context recognition|recognition=] would break fundamental aspects of
-the web. In many cases it's possible to change the design in a way that avoids
-the violation without breaking valid use cases, but for cases where that's not
-possible we refer you to [[[Privacy-Threat]]] which discusses tradeoffs in
-detail.
+When a [=user agent=] does have better information, it should adjust its
+[=partitions=] accordingly, for instance by partitioning identities per
+subdomain or site path when the structure of a given site is known.
```
Comment (jyasskin) [inline index.html:1099]: Here and up in this section is just generally shorter.

File: index.html [hunk 4]
```diff
@@ -1093,49 +1094,16 @@
...
-intend to be recognized.
-
-If a [=person=] visits two or more web pages from different [=partitions=],
-and does not explicitly express the same identity on each visit, the user agent should
-take steps to prevent [=same-site recognition=]. Note that sites can do harm
-even if they can't be *completely* certain that visits come from the same
-person. Further, note that sites must not assume that a person intends to be
-recognised across different sites simply because they are using the same
-identifier (such as an email address) with both or are relying on the same
-single sign-on identity.
```
Comment (jyasskin) [inline index.html:1112]: Unified with the cross-context recognition comment above.

File: index.html [hunk 5]
```diff
@@ -1093,49 +1094,16 @@
...
-* Disclose the least amount of identifying information, e.g.:
-  * Only share the parts of someone's profile information that is needed by
-  the website.
-  * Minimize the shared data; for example if a website requires age for
-  verification, then share that the
-  user is older than the minimum age, as opposed to specific age or date of
-  birth.
-* Support the user's intent to present a directed identity or a site-specific
-identity. E.g., if the website requires an email, then the user may choose a
-directed identifier without disclosing their real email.
```
Comment (jyasskin) [inline index.html:1131]: Seems redundant with the data minimization section.

File: index.html [hunk 6]
```diff
@@ -1093,49 +1094,16 @@
...
-
-This principle is limited in cases where preventing
-[=cross-context recognition|recognition=] would break fundamental aspects of
-the web. In many cases it's possible to change the design in a way that avoids
-the violation without breaking valid use cases, but for cases where that's not
```
Comment (jyasskin) [inline index.html:1136]: Lost its context in an earlier rewrite, and it didn't seem worth re-adding the context.

File: index.html [hunk 7]
```diff
@@ -1093,49 +1094,16 @@
...
-possible we refer you to [[[Privacy-Threat]]] which discusses tradeoffs in
-detail.
```
Comment (jyasskin) [inline index.html:1138]: This wound up in a paragraph that started "Where possible".

**Review by darobin — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (npdoty) [reply index.html:1131]: I'm not sure we want to remove this. Supporting direct identifiers and clarity about identity being presented both seem like specific affirmative recommendations that we want to make.

File: index.html [hunk 8]
```diff
@@ -1040,6 +1042,12 @@
 
 [=Cross-context recognition=] is only [=appropriate=] when the person being [=recognized=]
 can reasonably expect that recognition to happen and can control whether it does.
+Note that a person can use a piece of identifying information in two different
+contexts (e.g. their email or phone number) without that implying that they're
+using the same identity in both contexts. Unless there's some other indication
+that they intended to use a single identity, it is [=inappropriate=] to
+[=recognize=] them using that information.
+
```
Comment (npdoty) [inline index.html:1050]: It seems like the inappropriateness is not just recognizing, but requesting specific cross-origin PII for the purpose of recognition.

Maybe:
> unless ..., it is inappropriate to seek identifying information for cross-context recognition, or to recognize them using such information.

File: index.html [hunk 9]
```diff
@@ -1040,6 +1042,12 @@
 
 [=Cross-context recognition=] is only [=appropriate=] when the person being [=recognized=]
 can reasonably expect that recognition to happen and can control whether it does.
+Note that a person can use a piece of identifying information in two different
+contexts (e.g. their email or phone number) without that implying that they're
+using the same identity in both contexts. Unless there's some other indication
```
[the remark this answers is not in the document]
Comment (jyasskin) [reply index.html:1047]: Some of those characteristics are the websites the identity has used and the interests it has exposed, so it's easy to say the identities are different. We have to stretch a bit to say that an identity is the _same_ across websites, but if you intentionally log into one site by proving that you control an account on a different site, I think it's plausible to say that you're also presenting the characteristics you've shown to each site to the other one.

File: index.html (hunk 5 above)
Comment (jyasskin) [reply index.html:1131]: I think direct identifiers fit into the data minimization section, so I'll add them there. Clarity about the identity being presented is very hard to accomplish, except in some special circumstances like some uses of the Credential Management API, but I'll re-add it to this section as a nice-to-have.

File: index.html (hunk 8 above)
Comment (npdoty) [reply index.html:1051]: That sounds good, thanks.

File: index.html [hunk 10]
```diff
@@ -1074,14 +1078,12 @@
...
-express that they intend to change [=identities=], so the user agent can't enforce
-that sites actually separate [=identities=] at these boundaries.
-
-Instead, [=user agents=] may have to make some assumptions about the borders
-between [=contexts=]. By default, [=user agents=] define a
-<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+A <dfn>partition</dfn> is the [=user agent=]'s attempt to match how its user
+would understand a [=context=]. [=User agents=] don't have a perfect
+understanding of how their users experience the sites they visit, so they're
+often forced to approximate the boundaries between [=contexts=] when building
```
Comment (rhiaro) [inline index.html:1084]:
```suggestion
understanding of how their users experience the sites they visit, so they
often approximate the boundaries between [=contexts=] when building
```

Comment (jyasskin) [inline index.html:1084]: Christine suggested 
```suggestion
understanding of how their users experience the sites they visit, so they
often need to approximate the boundaries between [=contexts=] when building
```

File: index.html [hunk 11]
```diff
@@ -1093,49 +1095,21 @@
...
-
-This principle is limited in cases where preventing
-[=cross-context recognition|recognition=] would break fundamental aspects of
-the web. In many cases it's possible to change the design in a way that avoids
-the violation without breaking valid use cases, but for cases where that's not
-possible we refer you to [[[Privacy-Threat]]] which discusses tradeoffs in
-detail.
+When a [=user agent=] does have better information, it should adjust its
+[=partitions=] accordingly, for instance by partitioning identities per
+subdomain or site path when the structure of a given site is known.
```
Comment (jyasskin) [inline index.html:1100]: If I've captured Pete's suggestion correctly, we agree that

```suggestion
When a [=user agent=] knows that a site includes multiple contexts, it
should adjust its [=partitions=] accordingly, for instance by partitioning
identities per subdomain or site path.
```

works for everyone.

File: index.html [hunk 12]
```diff
@@ -1074,14 +1078,12 @@
...
-between [=contexts=]. By default, [=user agents=] define a
-<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+A <dfn>partition</dfn> is the [=user agent=]'s attempt to match how its user
+would understand a [=context=]. [=User agents=] don't have a perfect
+understanding of how their users experience the sites they visit, so they
+often need to approximate the boundaries between [=contexts=] when building
+[=partitions=]. In the absence of better information, a [=partition=] can be
+defined as:
 
 * A set of [=environments=] (roughly [^iframe^]s (including cross-site [^iframe^]s),
```
Comment (jyasskin) [inline index.html:1088]:
```suggestion
* A set of [=environments=] (roughly same-site and cross-site [^iframe^]s,
```

**Review by torgo — APPROVED** [review]
lgtm

File: index.html [hunk 13]
```diff
@@ -1074,16 +1078,14 @@
...
-<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+A <dfn>partition</dfn> is the [=user agent=]'s attempt to match how its user
+would understand a [=context=]. [=User agents=] don't have a perfect
+understanding of how their users experience the sites they visit, so they
+often need to approximate the boundaries between [=contexts=] when building
+[=partitions=]. In the absence of better information, a [=partition=] can be
+defined as:
 
-* A set of [=environments=] (roughly [^iframe^]s (including cross-site [^iframe^]s),
+* A set of [=environments=] (roughly same-site and cross-site [^iframe^]s,
```
Comment (jyasskin) [inline index.html:1088]:
```suggestion
* a set of [=environments=] (roughly same-site and cross-site [^iframe^]s,
```

**Pull request merged**